src/App.tsx

```diff
@@ -10,32 +11,38 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState({
+    title: 'Tab 1',
+    content: 'Some text 1',
+  });
+
+  const onTabSelected = (idTab: string) => {
```

Comment:
1) you can move handleTabSelect logic to <Tabs> component
2) you will invoke setSelectedTab and pass the full tab